tests/test_storages.py

```diff
@@ -71,6 +73,29 @@ def test_json_readwrite(tmpdir):
     assert get('A short one') is None
 
 
+def test_create_dirs():
+    temp_dir = tempfile.gettempdir()
+    db_dir = ''
+    db_file = ''
+
+    while True:
```

Comment:
Using pytest's tmpdir would allow us to not have to retry directory creation. Anyways, it's not a big deal really.